src/app/models/trip.js

```diff
@@ -0,0 +1,51 @@
+import Backbone from 'backbone';
+
+const Trip = Backbone.Model.extend({
+  url: function() {
```

Comment:
As I said earlier, you could also replace this `url` function with:

`urlRoot: https://ada-backtrek-api.herokuapp.com/trips/`

urlRoot is the endpoint that the API begins and Backbone can infer RESTful routes from there.